react-cors-spa/src/App.jsx

```diff
@@ -33,14 +34,15 @@ const queryClient = new QueryClient();
 function App() {
   return (
     <QueryClientProvider client={queryClient}>
-    <ThemeProvider theme={theme} defaultMode="dark">
+    <ThemeProvider theme={theme}>
     <CssBaseline />
       <div className="App">
         <header className="App-header">
           {APIEndPoint.startsWith("http") && <APIResult />}
         </header>
       </div>
       <div>
+        <AddProduct />
```

Comment:
Since App.jsx is supposed to be our top level component that **only** sets up all of our global items like theme and react query, we need to wrap these two components in another so that App.jsx has only one child that render UI.